### PR TITLE
chore: unify ports with start and start_dev command

### PR DESCRIPTION
**Pull request by obdulia-losantos**
This will simplify the bootstrap setup too

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,32 +128,19 @@ If the `nginx` container is also started the url ports can be removed.
...
 
-The client services are:
-
-  - **Gather & Aether (local)**  for `gather.local`.
-
-
-Other options are to log in via token, via basic authentication or via the
-standard django authentication process in the admin section.
-The available options depend on each container.
+The other option is the standard django authentication.
```
Comment (admbtlr) [inline README.md:167]: Please add "basic auth":

"The other option is the standard django authentication using basic auth."

File: app/gather/settings.py [hunk 2]
```diff
@@ -173,15 +173,15 @@
...
+        # CAS apps
         'django_cas_ng',
         'ums_client',
     ]
     AUTHENTICATION_BACKENDS += [
         'ums_client.backends.UMSRoleBackend',
     ]
 else:  # pragma: no cover
-    logger.info('No UMS enable!')
+    logger.info('No CAS enable!')
```
Comment (admbtlr) [inline app/gather/settings.py]: "enabled"

File: docker-compose-base.yml [hunk 3]
```diff
@@ -203,6 +187,7 @@ services:
     volumes:
       - ./app:/code
       - ./conf:/code/conf
+      # - ./.persistent_data/gather/static:/var/www/static
```
Comment (admbtlr) [inline docker-compose-base.yml]: do we need this line?

File: README.md (hunk 1 above)
Comment (obdulia-losantos) [reply README.md:167]: Not in Gather, basic auth is in ODK with the OpenRosa endpoints. In Gather we have the CAS option or the Django one.

File: docker-compose-base.yml (hunk 3 above)
Comment (obdulia-losantos) [reply docker-compose-base.yml]: Only when you use the `start` command, to share the volume with the nginx container.

File: README.md (hunk 1 above)
Comment (admbtlr) [reply README.md:167]: True, sorry, my mistake

File: docker-compose-base.yml (hunk 3 above)
Comment (admbtlr) [reply docker-compose-base.yml]: Can you put in a comment to explain that, and under what situations you would uncomment that line?

**Pull request merged**